api/python/arrus/ops/us4r.py

```diff
@@ -215,6 +216,15 @@ def get_sample_range_unique(self):
             raise ValueError("All TX/RXs should acquire the same sample range.")
         return next(iter(sample_range))
 
+    def get_subsequence(self, start, end):
```

Comment:
The same named function with same dosctring is defined in api/python/arrus/ops/imaging.py. Is it intencional?

Reply:
Yes, that's intentional, be cause we want to be able to both limit the SimpleTxRxSequence and the TxRxSequence to sub-sequence.